Resources/Prototypes/Entities/Mobs/Player/humanoid.yml

```diff
@@ -785,6 +785,9 @@
       rules: ghost-role-information-disaster-victim-rules
       raffle:
         settings: default
+      requirements:
```

Comment:
There should be a DeltaV comment indicating the requirement has been added compared to the original code at the first line, not just one saying the amount of hours.